usaspending_api/api_contracts/contracts/v2/references/naics.md

```diff
@@ -0,0 +1,45 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# NAICS [/api/v2/references/naics/]
+
+These endpoints are used to power USAspending.gov's NAICS search component on the advanced search page.
+
+
+## GET [/api/v2/references/naics/]
+
+This endpoint returns a list of Tier 1 NAICS codes, their descriptions, and their children
```

Comment:
`and their children` should probably read `and the count of all their children` @emily-brents